ReleaseProcessScript/tools/PesterTester/semver_functions.Tests.ps1

```diff
@@ -54,42 +54,42 @@ Describe "semver_functions" {
     }
 
     It "Get-Possible-Next-Versions-Develop_WithInvalidVersion_ShouldThrowException" {
-      {Get-Possible-Next-Versions-Develop "completelyWrongVersion" } | Should Throw "Your version 'completelyWrongVersion' does not have a valid format (e.g. 1.2.3-alpha.1)"
-      {Get-Possible-Next-Versions-Develop "1.2.3.4" } | Should Throw "Your version '1.2.3.4' does not have a valid format (e.g. 1.2.3-alpha.1)"            
-      {Get-Possible-Next-Versions-Develop "1.2.3-somethinginvalid.4" } | Should Throw "Your version '1.2.3-somethinginvalid.4' does not have a valid format (e.g. 1.2.3-alpha.1)"            
-      {Get-Possible-Next-Versions-Develop "1.2.3.alpha-4" } | Should Throw "Your version '1.2.3.alpha-4' does not have a valid format (e.g. 1.2.3-alpha.1)"            
-      {Get-Possible-Next-Versions-Develop "1.2.3.rc.4" } | Should Throw "Your version '1.2.3.rc.4' does not have a valid format (e.g. 1.2.3-alpha.1)"            
+      { Get-Possible-Next-Versions-Develop "completelyWrongVersion" } | Should Throw "Your version 'completelyWrongVersion' does not have a valid format (e.g. 1.2.3-alpha.1)"
+      { Get-Possible-Next-Versions-Develop "1.2.3.4" } | Should Throw "Your version '1.2.3.4' does not have a valid format (e.g. 1.2.3-alpha.1)"
+      { Get-Possible-Next-Versions-Develop "1.2.3-somethinginvalid.4" } | Should Throw "Your version '1.2.3-somethinginvalid.4' does not have a valid format (e.g. 1.2.3-alpha.1)"
+      { Get-Possible-Next-Versions-Develop "1.2.3.alpha-4" } | Should Throw "Your version '1.2.3.alpha-4' does not have a valid format (e.g. 1.2.3-alpha.1)"
+      { Get-Possible-Next-Versions-Develop "1.2.3.rc.4" } | Should Throw "Your version '1.2.3.rc.4' does not have a valid format (e.g. 1.2.3-alpha.1)"
     }
   }
 
-  Context "Get-Possible-Next-Versions-Support" {
-      It "Get-Possible-Next-Versions-Support_WithAlpha_ShouldReturnArray" {
-        $Version = "1.2.0-alpha.4"
-        $NextVersions = "1.2.0-alpha.5", "1.2.0-beta.1", "1.2.0"
+  Context "Get-Possible-Next-Versions-Hotfix" {
```

Comment:
typicallly, the hotfix-branches would not have patch-version = 0 given that patch-version = 0 is normally released from the master and used as the base for the support branch. (unless there's a supportbranch off a supportbranch). would it be helpful for reading the code if we updated the version numbers to reflect this?

Reply:
I noticed the diagram in https://re-motion.atlassian.net/browse/RMSRCBUILD-80 might have a flaw:

we start support/1.3 off support/1.2.1 and then immediately go to hotfix/1.3.1 instead of hotfix/1.3.0. Of course, strictly speaking, v1.3.0 is not a hotfix, but that could be a detail that's not relevant. we should however ensure that a support branch based off a support branch can start with patch-version = 0.